settings.py

```diff
@@ -1157,6 +1157,8 @@ def _pkce_required(client_id):
 
 IS_SAAS_ENVIRONMENT = SERVER_ENVIRONMENT in ('production', 'staging')
 
+IS_INDIA_ENVIRONMENT = SERVER_ENVIRONMENT == 'india'
```

Comment:
@dannyroberts Is it okay to have a variable `IS_INDIA_ENVIRONMNET` or we should have some setting variable for `production`, `staging` and `india`. If yes, any suggestions on what we should be calling it?

Reply:
I think it would make more sense to have a variable called something like `ALLOW_MAKE_SUPERUSER_COMMAND` set to `True` here and then set to `False` for production, staging, india in commcare-cloud. (You'll need to edit their `environments/*/public.yml` as well as `localsettings.py.j2`.)

Reply:
Just wanted to make sure you saw my comment here. It's not urgent, but I do think that it's still useful cleanup to do. If we apply this to another environment or decide not to apply it on staging, it makes much more sense to edit the respective environment than to have to come back to commcare-hq and edit `settings.py` and `make_superuser.py`

Reply:
Thanks, @dannyroberts for the reminder. I will clean it up by the weekend.

Reply:
@dannyroberts I have raised the PRs for the cleanup
https://github.com/dimagi/commcare-hq/pull/30107, https://github.com/dimagi/commcare-cloud/pull/4832